### PR TITLE
Adding TP feature table and inclusive language section

### PR DESCRIPTION
**Pull request by Preeticp**
This PR adds a TP table for features in Pipelines that are not yet GA. It also adds the inclusive language sections.
Aligned team: Dev Tools
Versions: 4.7 and 4.8
SME/QE review: Savita/Pavol
Peer Review: Souvik - done
Direct Preview link: https://deploy-preview-31975--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#technology-preview-features-1-4_op-release-notes

### CODE REVIEW

**Comment by sounix000** [comment]
\lgtm

**Comment by Preeticp** [comment]
Please note there are no changes made to the GitOps section, only the TP feature section is moved below the New features section.

**Review by sounix000 — APPROVED** [review]

File: modules/op-release-notes-1-4.adoc [hunk 1]
```diff
@@ -80,6 +80,30 @@ fsGroup:
...
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+.Technology Preview tracker
+[cols="1,1,1,1",options="header"]
+|===
+| Feature | OCP 4.5 |OCP 4.6 | OCP 4.7
```
Comment (siamaksade) [inline modules/op-release-notes-1-4.adoc:99]: @preethit shouldn't the version column refer to OpenShift Pipelines versions?

Comment (Preeticp) [reply modules/op-release-notes-1-4.adoc:99]: @siamaksade, this table is under the [1.4 RN section](https://docs.openshift.com/container-platform/4.7/cicd/pipelines/op-release-notes.html#op-release-notes-1-4_op-release-notes), not at the top unlike the compatibility matrix, hence the OCP versions. On second thought, I wonder if we are repeating the same things in multiple places - compatibility matrix the note on TP features and the TP table. I will work on consolidating this information and run it by you.

BTW my GH ID is @Preeticp.  preetihit is another person on the OCP team :)

Comment (siamaksade) [reply modules/op-release-notes-1-4.adoc:99]: @Preeticp Sorry for the mixup :)

**Pull request merged**